curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/complete-a-promise-with-resolve-and-reject.english.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 5cdafbc32913098997531680
+title: Complete a Promise with resolve and reject
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+A promise has three states: <code>pending</code>, <code>fulfilled</code>, and <code>rejected</code>. The promise you created in the last challenge is forever stuck in the <code>pending</code> state because you did not add a way to complete the promise. The <code>resolve</code> and <code>reject</code> parameters given to the promise argument are used to do this. <code>resolve</code> is used when you want your promise to succeed, and <code>reject</code> is used when you want it to fail. These are methods that take an argument, as seen below.
+
+```js
+const myPromise = new Promise((resolve, reject) => {
+  if(true) {
```

Comment:
```
  if(true) {
```

The promise here will always resolve. I'm wondering if maybe we should add a note here explaining that this is just an example, otherwise some campers might get the idea that this is how promises are always resolved?

Just a thought...

Reply:
I don't think it's necessary here, this is just a code example. Let me know if you still think something needs to be added.

I suppose I could change `true` to `condition` or something...
`if(condition here)`

Reply:
Without an explanation about `if(true)`, there's a very good chance of someone completely new to Promises interpreting this as literally how promises work. Moreover, this code is in the challenge description (not the challenge code), so again, one can never know for sure if this is just an example. 

This is fine though - `if(condition here)`.